simulator/mips/mips_instr.cpp

```diff
@@ -498,6 +526,16 @@ void MIPSInstr::set_v_dst( uint32 value)
     }
 }
 
+uint64 MIPSInstr::get_v_src2() const
+{
+    switch ( operation)
+    {
+        case OUT_I_STORE64L: return static_cast<uint64>( v_src2) & 0xFFFFFFFFFFFF0000;
```

Comment:
Are you guys sure that it does StoreL/StoreR correctly? Offset should be dependent on `v_src1 % 8` if I understand it correctly.
I assume you make skip `ldl`, `ldr`, `sdr`, `sdl` as we have a separate bug tracker for them (including `lwr`, `lwl`, `swl`, `swd`)